qdb/server/session_store.py

```diff
@@ -243,7 +238,7 @@ def attach_tracer(self, uuid, socket, local_pid, pause_signal):
             # Signal to the tracer that no client attached.
             self._send_to_socket(socket, fmt_err_msg(
                 'client', 'No client',
-                serial=pickle.dumps
```

Comment:
python 3 pickle uses a different protocol than 2 by default. To make this more simple, we are going to use json for all communication instead of a mix